src/Entity/CitizenProject.php

```diff
@@ -364,6 +350,10 @@ public function update(
         if (null === $this->postAddress || !$this->postAddress->equals($address)) {
             $this->postAddress = $address;
         }
+
+        if (!$this->phone->equals($phone)) {
```

Comment:
What is the usefulness of this ?

Reply:
Persisting a clone while letting an orphan behind.